analysis/deep2_r23_o32/plotting/plot_difference_curvefit.py

```diff
@@ -85,7 +85,7 @@ def plot_difference_threevariable(lR23, lO32, OH, lO32_all, pdf_file,
                            (lO32[nn] <= y_ii_max))[0]
 
             ii_label = ''
-            if label =='DEEP2':  # n_sample-1:
+            if label == 'DEEP2':  # n_sample-1:
```

Comment:
I'm a bit confused. If label is a list, how is it that you're comparing it to a `str`. Maybe I'm missing something.